### DISCUSSION TITLE
Copy things in Excel  and can't be pasted in luckysheet

### DISCUSSION THREAD

**Discussion by huweige** [dream-num/Luckysheet#591] [Q&A]
1. Select several cells from luckysheet to copy

2. Then paste it into Microsoft Excel

3. Copy things in Excel and paste them to luckysheet. At this time, you will find that they can't be pasted in.

If there is no copy operation in luckysheet, the contents in Excel can be copied to luckysheet

**Comment by mikeungers** [comment]
Potential bugfix:
https://github.com/mikeungers/Luckysheet/commit/0f7d467c3164c5effc3f185bceb7afc94b7a93c5

**Comment by PedroFerreiraBento** [comment]
Same problem here, trying to develop a solution...

**Comment by p1nkv4lkyr13** [comment]
I got the same issue. Luckysheet won't take values from Excel if any cells have border. 
The issue is probably in this line [controllers/selection.js - line 683](https://github.com/dream-num/Luckysheet/blob/418a210ffd69183551c7bffcd5a1b674090cd3b8/src/controllers/selection.js#L683).
cfg["borderInfo"] is not null, it's an empty object.
